### ISSUE TITLE
Getting error "Cannot read property 'setAttribute' of undefined"

### ISSUE THREAD

**Issue by pbadenski** [ampproject/worker-dom#138]
This issue can be quite overwhelming, happened to me when trying to use worker-dom with a pretty large library (https://github.com/handsontable/handsontable).

Error stack:

```
Uncaught TypeError: Cannot read property 'setAttribute' of undefined
    at Object.0 (mutator.js:69)
    at MUTATION_QUEUE.forEach.mutation (mutator.js:134)
    at Array.forEach (<anonymous>)
    at syncFlush (mutator.js:133)
    at requestAnimationFrame (mutator.js:122)
```

Code in mutator.js (Line 69 highlighted with a comment):
```
[1 /* CHARACTER_DATA */](mutation, target) {
        const value = mutation[18 /* value */];
        if (value) {
            // Sanitization not necessary for textContent.
            target.textContent = getString(value);
        } // **Line 69**
    },
```

Seems that `target` is `undefined`.  I added some debugging code to `storeNodes` and it never stores the node with `id` that is being requested. `id` requested is 28 & `storeNodes` only gets called starting with 68.

**Comment by pbadenski** [comment]
I spent a bit more time debugging this. Based on my relatively shallow and mostly intuitive understanding I created a PoC fix: https://github.com/pricingmonkey/worker-dom/commit/146361baec36767971174d534fab854166898794

I think it might be that creating detached nodes might be causing an issue, since it was only document.body that was passed in during hydration. I extended hydration from single element to array of nodes & added passing of all nodes. It solved this issue.

P.S. Once this is resolved handsontable (which is quite complex JS library) rerenders via worker-dom! This project is absolutely amazing :D

I know this might not be the right place... I'm extremely excited seeing how nicely the UI rerendering gets offloaded to the webworker, so let me share this:

<img width="1439" alt="screen shot 2018-10-31 at 17 36 19" src="https://user-images.githubusercontent.com/68713/47807295-a2d8fc00-dd33-11e8-96f1-8abfe0fbe80b.png">

**Comment by kristoferbaxter** [comment]
That is so awesome to see!

We have an open PR #135 which likely fixes the issue you're seeing as well.

**Comment by kristoferbaxter** [comment]
@pbadenski – We just released `0.2.0` which includes #135. Can you test this new version with your application and let us know if it resolves the issue?

**Comment by kristoferbaxter** [comment]
That's great news, going to close this issue.
